proposals/function-pointers.md

```diff
@@ -169,7 +178,7 @@ In an unsafe context, a method `M` is compatible with a function pointer type `F
 - For each `ref`, `out`, or `in` parameter, the parameter type in `M` is the same as the corresponding parameter type in `F`.
 - If the return type is by value (no `ref` or `ref readonly`), an identity, implicit reference, or implicit pointer conversion exists from the return type of `F` to the return type of `M`.
 - If the return type is by reference (`ref` or `ref readonly`), the return type and `ref` modifiers of `F` are the same as the return type and `ref` modifiers of `M`.
-- The calling convention of `M` is the same as the calling convention of `F`.
+- The calling convention of `M` is the same as the calling convention of `F`. This includes both the calling convention bit, as well as any calling convention flags specified in the unmanaged identifier.
```

Comment:
>This includes both the calling convention bit, as well as any calling convention flags specified in the unmanaged identifier. [](start = 78, length = 125)

I think this doesn't quite match the last proposal discussed on the internal thread. We need to specify precise rules for determining if calling conventions are compatible. Also, I am not sure if talking about that in syntactic terms ("calling convention flags specified in the unmanaged identifier") is the right approach.